tools/postgres-init-dev-db

```diff
@@ -24,7 +24,7 @@ if [[ $# == 0 ]]; then
     USERNAME=zulip
     PASSWORD=$("$(dirname "$0")/../bin/get-django-setting" LOCAL_DATABASE_PASSWORD)
     DBNAME=zulip
-    SEARCH_PATH="$USERNAME",public
+    SEARCH_PATH="$USERNAME",public,pgroonga,pg_catalog
```

Comment:
Hmm, this is tricky -- `postgres-init-db` is run before we give the user the opportunity to put things into settings.py.  Let me think about this a bit more and get back to you.  It might work to move `postgres-init-db` to being run a bit later in the installation process.